plugins/by-name/rust-tools/default.nix

```
@@ -97,39 +97,41 @@ in
         ]
       ] "rust-tools hover window" "";
 
-      maxWidth = helpers.defaultNullOpts.mkUnsignedInt null "Maximal width of the hover window. null means no max.";
+      maxWidth = lib.nixvim.defaultNullOpts.mkUnsignedInt null "Maximal width of the hover window. null means no max.";
 
-      maxHeight = helpers.defaultNullOpts.mkUnsignedInt null "Maximal height of the hover window. null means no max.";
+      maxHeight = lib.nixvim.defaultNullOpts.mkUnsignedInt null "Maximal height of the hover window. null means no max.";
 
-      autoFocus = helpers.defaultNullOpts.mkBool false "whether the hover action window gets automatically focused";
+      autoFocus = lib.nixvim.defaultNullOpts.mkBool false "whether the hover action window gets automatically focused";
     };
 
     crateGraph = {
-      backend = helpers.defaultNullOpts.mkStr "x11" ''
+      backend = lib.nixvim.defaultNullOpts.mkStr "x11" ''
         Backend used for displaying the graph
         see: https://graphviz.org/docs/outputs/
       '';
 
-      output = helpers.defaultNullOpts.mkStr null "where to store the output, nil for no output stored";
+      output = lib.nixvim.defaultNullOpts.mkStr null "where to store the output, nil for no output stored";
 
-      full = helpers.defaultNullOpts.mkBool true ''
+      full = lib.nixvim.defaultNullOpts.mkBool true ''
         true for all crates.io and external crates, false only the local crates
       '';
 
-      enabledGraphvizBackends = helpers.defaultNullOpts.mkNullable (types.listOf types.str) null ''
-        List of backends found on: https://graphviz.org/docs/outputs/
-        Is used for input validation and autocompletion
-      '';
+      enabledGraphvizBackends =
+        lib.nixvim.defaultNullOpts.mkNullable (lib.types.listOf lib.types.str) null
+          ''
+            List of backends found on: https://graphviz.org/docs/outputs/
+            Is used for input validation and autocompletion
+          '';
     };
 
     server = {
-      standalone = helpers.defaultNullOpts.mkBool true ''
+      standalone = lib.nixvim.defaultNullOpts.mkBool true ''
         standalone file support
         setting it to false may improve startup time
       '';
-    } // (import ../../lsp/language-servers/rust-analyzer-config.nix lib helpers);
+    } // (import ../../lsp/language-servers/rust-analyzer-config.nix lib);
```

If we're touching the line anyway...
```suggestion
    } // import ../../lsp/language-servers/rust-analyzer-config.nix lib;
```